packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -224,50 +274,91 @@ class SnackBar extends StatelessWidget {
     ];
     if (action != null) {
       children.add(ButtonTheme.bar(
-        padding: const EdgeInsets.symmetric(horizontal: _kSnackBarPadding),
+        padding: EdgeInsets.symmetric(horizontal: snackBarPadding),
         textTheme: ButtonTextTheme.accent,
         child: action,
       ));
     } else {
-      children.add(const SizedBox(width: _kSnackBarPadding));
+      children.add(SizedBox(width: snackBarPadding));
     }
-    final CurvedAnimation heightAnimation = CurvedAnimation(parent: animation, curve: _snackBarHeightCurve);
-    final CurvedAnimation fadeAnimation = CurvedAnimation(parent: animation, curve: _snackBarFadeCurve, reverseCurve: const Threshold(0.0));
-    Widget snackbar = SafeArea(
+    final CurvedAnimation heightAnimation =
```

Comment:
Looks like they prefer this and the variable underneath it to be on one line each even tho it's over 80.

Reply:
Done